doc/filters/language_name.rst

```diff
@@ -1,8 +1,8 @@
 ``language_name``
 =================
 
-The ``language_name`` filter returns the language name given its two-letter
-code:
+The ``language_name`` filter returns the language name given its ISO 639-1 (two-letter code)
```

Comment:
this description is incomplete. `language_name` accepts more than the ISO codes. Some codes corresponding to localized names are also supported.